projects/opendr_ws_2/src/opendr_perception/opendr_perception/object_detection_2d_detr_node.py

```diff
@@ -47,64 +52,168 @@ def __init__(self, input_rgb_image_topic="image_raw", output_rgb_image_topic="/o
         """
         super().__init__('object_detection_detr_node')
 
-        self.image_subscriber = self.create_subscription(ROS_Image, input_rgb_image_topic, self.callback, 1)
-
         if output_rgb_image_topic is not None:
-            self.image_publisher = self.create_publisher(ROS_Image, output_rgb_image_topic, 1)
+            self.image_publisher = self.create_publisher(ROS_Image, output_rgb_image_topic, 10)
```

Comment:
All queues should be set to 1 to avoid delays #275.
```suggestion
            self.image_publisher = self.create_publisher(ROS_Image, output_rgb_image_topic, 1)
```